tests/client.py

```diff
@@ -773,6 +781,111 @@ def create(self, stream, company_ids=[], subscriptions=[], times=1):
         else:
             raise NotImplementedError(f"There is no create_{stream} method in this dipatch!")
 
+    def delete_custom_contacts_property(self):
+        """ Clearup the custom contact properties before CRUD test """
+        custom_properties = ['custom_string', 'custom_number', 'custom_date', 'custom_datetime', 'multi_pick']
+        for property in custom_properties:
+             get_url = f"{BASE_URL}/properties/v1/contacts/properties/named/" + property
+             response = self.get_custom_property(get_url)
+             if ( response.status_code == 404 ):
+                 continue
+             url = f"{BASE_URL}/properties/v1/contacts/properties/named/" + property
+             response = self.delete(url)
```

Comment:
Consider using raise_for_status().  Should we re-try if we fail on a delete?

Reply:
hmm... raise_for_status will log it as an error. I don't know if we want that.